resources/styles/book-content/tips-for-success.less

```diff
@@ -16,25 +18,43 @@
     padding: 0 10px;
     margin: 0;
   }
-  > ul:first-child {
-    padding-top: 10px;
-    border-top: 1px solid @tutor-neutral-light;
+  p {
+    #fonts > .sans(1.7rem, 1.5em);
+    color: @tutor-neutral-dark;
+    padding: 15px 20px;
+    border-left: 1px solid @tutor-neutral-lighter;
+    border-right: 1px solid @tutor-neutral-lighter;
+    background: @tutor-neutral-lightest;
+    &:first-child {
```

Comment:
Oh, I see. That way the yellow "Tips for success" piece does not span the entire width? If that is the case should the styling just be `:first-child` rather than `p:first-child` (or "What happens when the first child is not a `<p>`?